content/docs/user-guide/build/distributable-engine.md

```diff
@@ -0,0 +1,226 @@
+```
+
+The `engine` field of the `project.json` file will now be the name of the engine that was set as `LY_ENGINE_VERSION_NAME` during configuration.
+
+## Perform a project build
+
+Whether you have created a new project or updated an existing one, in order to create your final distributable build, you need to build the project and the final engine binaries to check into source control. 
+
+From your project directory, configure and build the project.
+    
+```cmd
+cmake -B build/windows_vs2019 -S .
+cmake --build build/windows_vs2019 --config profile
+```
+
+## (Optional) Create and load an initial level
+
+To help creative teams who will be using the tools (but not building them), we recommend creating an initial level for them to load as a first step in working with the project.
+
+1. Launch your local build of the editor with the project loaded. 
+    
+    ```cmd
+    C:\o3de\bin\Windows\profile\Default\Editor.exe --project-path W:\MyProject
+    ```
+1. After the Asset Processor finishes and the Editor loads, if your project isn't already configured to load a level, you'll see a dialog instructing you to load or create a level. In this example, it's been named "MyLevel".
+2. Quit the Editor.
+3. Create or modify a registry file for your project at `<project-location>\Registry\autoexec.game.setreg`. For this example, it would be located at `W:\MyProject\Registry\autoexec.game.setreg`.
+   
+   Add the following to this file:    
+    ```
+    {
+        "O3DE": {
+            "Autoexec": {
+                "ConsoleCommands": {
+                    "LoadLevel": "MyLevel"
+                }
+            }
+        }
+    }
+    ```
+
+To verify that the level loads when the game is launched, run the project's game launcher:
+
+```cmd
+W:\MyProject\build\windows_vs2019\bin\profile\MyProject.GameLauncher.exe
+```
+
+## Create a distributable build
+
+The final step to create the binaries for distribution to project developers is to create the set of install binaries that they can use without building the engine from source. Like you did to create the local binaries for the engine, you'll now create binaries for the project.
+
+From the project directory, run the following steps:
+
+1. Create an install build of the project.
+    
+    ```cmd
+    cmake --build build\windows_vs2019 --config profile --target INSTALL
+    ```
+
+    This creates the binaries in an `install` subdirectory of your project. For our example it would be `W:\MyProject\install`.
+
+## Distribute your build
+
+At this point, the created `install` directory is your final set of distributables. It's up to your team to decide where to host the build, but we recommend a source control system that has good blob support.
+
+{{< important >}}
+The .gitignore rule automatically adds the following folders to ignore for git. If using another source control system, make sure the following subdirectories of your project are ignored and not checked in:
```

Comment:
It may be good to notate that the .gitignore file is located in the project root after project creation and can be referenced to determine the list of directories/files  to ignore